### ISSUE TITLE
source of sea regions?

### ISSUE THREAD

**Issue by LennertSchepers** [iobis/woa2-fish#1]
Hi @pieterprovoost 
Short question: is marineregions the source of the areas? or not?
Thanks

**Comment by pieterprovoost** [comment]
Yes, that's correct. I'll try to add some documentation, please leave open.
